common/api/bity.js

```diff
@@ -1,36 +1,38 @@
 // @flow
 import bityConfig from 'config/bity';
-import {combineAndUpper} from 'utils/formatters'
+import { checkHttpStatus, parseJSON } from './utils';
+import { combineAndUpper } from 'utils/formatters';
 
-
-function findRateFromBityRateList(rateObjects, pairName) {
+function findRateFromBityRateList(rateObjects, pairName: string) {
   return rateObjects.find(x => x.pair === pairName);
 }
 
-// FIXME better types
-function _getRate(bityRates, origin: string, destination: string) {
-  const pairName = combineAndUpper(origin, destination);
+function _getRate(bityRates, originKind: string, destinationKind: string) {
+  const pairName = combineAndUpper(originKind, destinationKind);
   const rateObjects = bityRates.objects;
   return findRateFromBityRateList(rateObjects, pairName);
 }
 
 /**
  * Gives you multiple rates from Bitys API without making multiple API calls
- * @param arrayOfOriginAndDestinationDicts - [{origin: 'BTC', destination: 'ETH'}, {origin: 'BTC', destination: 'REP}]
+ * @param transactionPairs - [{origin: 'BTC', destination: 'ETH'}, {origin: 'BTC', destination: 'REP}]
  */
-function getMultipleRates(arrayOfOriginAndDestinationDicts) {
+export type TransactionPair = {
+  originKind: string,
+  destinationKind: string
+};
```

Comment:
Will fix. Dunno how Flow isn't picking this stuff up, but then again the function is going unused for now.